deepfence_frontend/apps/dashboard/src/components/hosts-connector/K8ConnectorForm.tsx

```diff
@@ -24,7 +24,15 @@ const useGetApiToken = () => {
   });
 };
 
+const useGetVersion = () => {
+  return useSuspenseQuery({
+    ...queries.setting.productVersion(),
+    keepPreviousData: true,
```

Comment:
dont need `keepPreviousData` here, its only useful in queries where there is pagination.